### ISSUE TITLE
Boot time takes noticeably longer with v3.0

### ISSUE THREAD

**Issue by itsminht** [CelesteBlue-dev/PSVita-StorageMgr#39]
With the latest version (v3.0), when restarting or powering on Vita, the boot time is now considerably longer compared to using previous versions of storagemgr or having the plugin disabled. Upon boot, vita hangs on the henkaku/enso logo for about 5-10 seconds longer.

Not a big issue, difference is only a few seconds, just like to point this out in case it could be fixed. 

By the way, thank you for your hard work on v3.0 and fixing all the other issues!

**Comment by CelesteBlue-dev** [comment]
Please test :
https://github.com/xyzz/gamecard-microsd/releases

**Comment by itsminht** [comment]
I would definitely test it out if I could, but I'm the few people that uses my sd2vita mounted as uma0 and not ux0. The plugin you've linked me mounts sd2vita as ux0 by default, so I unfortunately can't use it.

If anyone else could test the boot times for the plugin linked above, that would be great!

**Comment by Madridi7** [comment]
@CelesteBlue-dev 
I can confirm storagemgr has a delay in boot time compared to gamesd. I noticed it since before the latest versions of gamesd and storagemgr were released.

I haven’t had a chance to try the latest version of either plugin yet, but from what I understand it’s more noticeable now and even longer time is being taken. But my point is, the problem definitely existed before the latest version, even if it was shorter (and therefore less noticeable)

Thanks for the plugin by the way. It’s one of my favorites :)

**Comment by Matsku12** [comment]
I also confirm the issue. Feels very long time when booting. I thought its SD2vita issue, but good to know that its caused by this so it might be fixed someday :)

**Comment by SCOTT0852** [comment]
3.68 h-encore takes longer to load too. It freezes for a while when I press Exit. It doesn't really bother me since I only reboot when I add a new plugin, but it's very noticable.

**Comment by Madridi7** [comment]
So is this abandoned now?

**Comment by RinMaru** [comment]
looks like itt gamesd is more updated and faster

**Comment by CelesteBlue-dev** [comment]
No it is not abandoned. I am back.

**Comment by RinMaru** [comment]
you should partner with [YAMT](https://github.com/SKGleba/yamt-vita) its very fast.

**Comment by CelesteBlue-dev** [comment]
Yes YAMT is one of the ways to improve speed, but not the only one I'm aware of. Soon.
